routes/users.js

```diff
@@ -0,0 +1,105 @@
+const express = require('express');
+const router = express.Router()
+const bcrypt = require('bcryptjs')
+const passport = require('passport')
+
+//User model
+const User = require('../models/User')
+
+//Login-page
+router.get('/login', (req,res)=>{
+    res.render('login')
+})
+
+//Register page
+router.get('/register', (req,res)=>{
+    res.render('register')
+})
+
+//Register handle
+router.post('/register', (req,res)=>{
+    const {name, email, password, password2} = req.body;
```

Comment:
This function should be in controller directory.

Reply:
updated